storage/volumes.md

```diff
@@ -512,12 +517,64 @@ $ docker volume create \
 	--name cif-volume
 ```
 
-Notice the `addr` option is required if using a hostname instead of an IP so docker can perform the hostname lookup.
+The `addr` option is required if you specify a hostname instead of an IP.
+This lets Docker perform the hostname lookup.
+
+### Mount a block storage device
+
+You can mount a block storage device, such as an external drive or a drive partition, to a container.
+The following example shows how to create and use a file as a block storage device,
+and how to mount the file as a container volume.
+
+> **Note**
+>
+> The following procedure is only an example. This example creates an `ext4`
+> filesystem and mounted into a container. The filesystem support of your
+> system depends on the version of the Linux kernel you are using.
+>
+> Under the hood, the `--mount` flag invokes the Linux `mount` command and forwards
+> the options you pass to it unaltered. Docker doesn't implement any additional
+> functionality on top of the native mount feature supported by the Linux kernel.
+
+1. Create a file and allocate some space to it:
+
+   ```console
+   $ fallocate -f 1G disk.raw
+   ```
+
+2. Build a filesystem onto the `disk.raw` file:
+
+   ```console
+   $ mkfs.ext4 disk.raw
+   ```
+
+3. Create a loop device:
+
+   ```console
+   $ losetup -f --show disk.raw
+   /dev/loop5
+   ```
+
+   > **Note**
+   >
+   > `losetup` creates an ephemeral loop device that's removed after
+   > system reboot, or manually removed with `losetup -d`.
```

Comment:
Maybe we should also add a final step which does the cleanup with `losetup -d /dev/loop5`?